llvm/test/CodeGen/AMDGPU/GlobalISel/postlegalizercombiner-and.mir

```diff
@@ -203,6 +203,7 @@ body:             |
     ; CHECK-LABEL: name: remove_and_65535_groupstaticsize
     ; CHECK: liveins: $vgpr0_vgpr1
     ; CHECK-NEXT: {{  $}}
+    ; CHECK-NEXT: %ptr:_(p1) = COPY $vgpr0_vgpr1
```

Comment:
I assume this copy was always here, and this test needs regeneration for some reason?

Reply:
Oh, this is just DCE disabled, but for the test purposes the copy shouldn't have been here in the first place